airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,7 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
-<div id="editor"></div>
+<div id="editor" data-slug="{{ slug }}" data-url="{{ url(popcorn:data) }}"></div>
```

Comment:
How could this have worked in testing? Perhaps we forgot to add a test that executes this view. 
It's supposed to be `{{ url('popcorn:data') }}`. 

Check your coverage first and then, write a test that executes this template (which should fail because the expression `{{ url(popcorn:data) }}` is most likely not valid in Jinja.